coalib/misc/Exceptions.py

```diff
@@ -1,15 +1,11 @@
-from pyprint.NullPrinter import NullPrinter
-
-from coalib.misc import Constants
-from coalib.output.printers.LogPrinter import LogPrinter
-
 from pkg_resources import VersionConflict
 
+from coalib.misc import Constants
+import coalib.output.printers.LogHelper as LogHelper
+from coalib.output.printers.LOG_LEVEL import LOG_LEVEL
 
-def get_exitcode(exception, log_printer=None):
-    log_printer = (LogPrinter(NullPrinter()) if log_printer is None
-                   else log_printer)
 
+def get_exitcode(exception):
```

Comment:
actually I wonder if we should move the log_exception method into this file here instead of having an own LogHelper